ext/session/session.c

```diff
@@ -2856,7 +2856,8 @@ static int php_rinit_session(zend_bool auto_start) /* {{{ */
 {
 	php_rinit_session_globals();
 
-	if (PS(mod) == NULL) {
+	PS(mod) = NULL;
```

Comment:
Do we need to also do this for PS(serializer) below?

Reply:
That's not necessary, since `OnUpdateSerializer` handles this (any unknown handler will set it to NULL). `OnUpdateSaveHandler` doesn't do this for `user` handlers (these are special, since they can only work for `session_set_save_handler()`.